main.pm

```diff
@@ -196,6 +196,11 @@ sub gnomestep_is_applicable() {
     return check_var("DESKTOP", "gnome");
 }
 
+sub snapper_is_applicable() {
+    my $fs = get_var("FILESYSTEM", 'btrfs');
+    return ( $fs eq "btrfs" && get_var("HDDSIZEGB") > 10);
```

Comment:
Just a note that we should be careful with this condition in case of autoyast tests (or when configuration depends on external source). That way `FILESYSTEM` may be not set, but the actual FS can be whatever.